### PR TITLE
Empty DOCKER_IMAGE_ORG produces valid image name

### PR DESCRIPTION
**Pull request by manno**
Not sure if this is required, but somehow I got this error:

```Jul 24 16:17:20 minikube kubelet[3452]: E0724 16:17:20.510911    3452 pod_workers.go:190] Error syncing pod 9a75de21-ad6c-11e9-809e-0800274aac91 ("cf-operator-test1563899338-1-4-66cfb4646b-cstdj_test1563899338-1-4(9a75de21-ad6c-11e9-809e-0800274aac91)"), skipping: failed to "StartContainer" for "cf-operator" with InvalidImageName: "Failed to apply default image tag \"/cf-operator:0.1-dev\": couldn't parse image reference \"/cf-operator:0.1-dev\": invalid reference format"```

### CODE REVIEW

File: pkg/bosh/converter/docker_image.go [hunk 1]
```diff
@@ -13,6 +13,9 @@ type DockerSource struct {
 // GetName returns the name of the docker image
 // More info: https://kubernetes.io/docs/concepts/containers/images
 func (d DockerSource) GetName() string {
+	if d.Organization == "" {
+		return d.Repository + ":" + d.Tag
+	}
```
Comment (andreas-kupries) [inline pkg/bosh/converter/docker_image.go]: Is this testable ? (I see no unit tests for docker_image.go at all)

Comment (manno) [reply pkg/bosh/converter/docker_image.go]: Sure, feel free to add some. Otherwise I can backfill them tomorrow.

**Comment by manno** [comment]
This is actually failing several unit tests, because of https://github.com/cloudfoundry-incubator/cf-operator/blob/master/pkg/bosh/converter/bpm_resources_test.go#L142

**Comment by manno** [comment]
This is actually failing several unit tests, because of https://github.com/cloudfoundry-incubator/cf-operator/blob/master/pkg/bosh/converter/bpm_resources_test.go#L142


I'm closing this PR and I did add a story to investigate this further: https://www.pivotaltracker.com/story/show/167509239

**Pull request closed without merging**